samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ * @brief Wi-Fi Raw Tx Packet sample
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(raw_tx_packet, CONFIG_LOG_DEFAULT_LEVEL);
+
+#include <nrfx_clock.h>
+#include <zephyr/kernel.h>
+
+#include "net_private.h"
+
+int try_wifi_connect(void);
```

Comment:
please move to a header file.

Reply:
We don't ahve specific header for the sample. Is that okay if I write a header file only for this declaration?

Reply:
Sure, it can act as a placeholder.